lib/order.rb

```diff
@@ -0,0 +1,21 @@
+class Order
+
+	attr_reader :dishes
+
+	def initialize(menu)
+		@dishes = {}
+		@menu = menu
+	end
+
+	def add(dish, quantity)
+		raise NoItemError, "#{dish} is not on the menu" unless menu.has_dish?(dish)
```

Comment:
Style/SignalException: Use `fail` instead of `raise` to signal exceptions. (https://github.com/bbatsov/ruby-style-guide#fail-method)